exercise.txt

```diff
@@ -0,0 +1,34 @@
+// 2 3 5
+// 3 5 6
+// 5 6 2
+// 6 2 3 
+
+
+// { 2, 3, 5, 6 };
+
+void func(int* a, int* b, int size) {
+    for (int i = 0; i < size; ++i) {
+        int tmp = 1;
+        for (int j = 0; j < size;++j) {
```

Comment:
Here you use 2 nested for loops which time complexity is O(n^2), you can reduce it